src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -253,6 +255,12 @@ internal static class Driver
         internal static readonly OptionInfo<Uri?> BaseUriOption = new OptionInfo<Uri?>(
             ImmutableList.Create("--base-uri"), default, "The base URI of the Azure Quantum endpoint.");
 
+        /// <summary>
+        /// The base URI option.
+        /// </summary>
+        internal static readonly OptionInfo<string?> LocationOption = new OptionInfo<string?>(
+            ImmutableList.Create("--location"), default, "The location to use with the default endpoint.");
```

Comment:
You should indicate what the default is (westus) in the help message, or does that happen automatically?

Reply:
Since in principle the driver itself is agnostic about the default endpoint, I think it should also be agnostic about the default location. I added the default location and endpoint in the `Constants` class in `src/Azure/Azure.Quantum.Client/Utility/Constants.cs`.